keepalived/core/global_data.c

```diff
@@ -163,7 +163,9 @@ alloc_global_data(void)
 		new->enable_snmp_checker = true;
 #endif
 	}
+#ifdef _WITH_LVS_
```

Comment:
This is inside a conditional compilation block `#ifdef _WITH_SNMP_`, but it is needed regardless of whether it is being build with SNMP. In my updated patch, I have moved it outside the _WITH_SNMP_ block.